pkgs/by-name/pr/protoc-gen-lint/package.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "protoc-gen-lint";
+  version = "0.3.0";
```

Comment:
Since this release is quite old I'd be tempted to use the latest commit

https://github.com/ckaznocha/protoc-gen-lint/compare/v0.3.0...master

But also there's no obvious worries like `x/net` so may not be worth it

Reply:
Yeah that's fine. It's only really necessary when there's significant dependency rot or vulns :slightly_smiling_face: